### DISCUSSION TITLE
Invitation from Unofficialsf.com

### DISCUSSION THREAD

**Discussion by alexed1** [Flow-Scanner/lightning-flow-scanner#265] [Ideas]
Hi, Reuben. This looks interesting! I have a website called unofficialsf.com and would be interested in publishing a blog post with a video that introduces your plugin, if you'd like to write one. You can reach me at alex@edelstein.org

**Comment by RubenHalman** [comment]
Hi @alexed1, would you be open to reconsider doing this? I see new opportunity in discussing the GitHub Action and created a [PR](https://github.com/UnofficialSF/LightningFlowComponents/pull/1686) to implement it in UnofficialSF!=) We would really appreciate your consideration!
